datapackage.json

```diff
@@ -21,11 +21,12 @@
     }
   ],
   "version": "2.11.0",
-  "created": "2024-12-06T16:14:38.044099+00:00",
+  "created": "2024-12-11T16:26:59.665471+00:00",
```

Comment:
@dsmedia make sure to regenerate this file to get the changes since (1ff024c5225e409d17f8e8435a0b1af6b71182d2)